app/containers/ReactionsList.tsx

```diff
@@ -0,0 +1,148 @@
+}
+
+interface IReactionsListProps extends IReactionsListBase {
+	reactions?: IReaction[];
+}
+
+interface ITabBarItem extends IReactionsListBase {
+	tab: IReaction;
+	index: number;
+	activeTab?: number;
+	goToPage?: (index: number) => void;
+}
+interface IReactionsTabBar extends IReactionsListBase {
+	activeTab?: number;
+	tabs?: IReaction[];
+	goToPage?: (index: number) => void;
+}
+
+const TabBarItem = ({ tab, index, goToPage, activeTab, baseUrl, getCustomEmoji }: ITabBarItem) => {
+	const { colors } = useTheme();
+	return (
+		<Pressable
+			key={tab.emoji}
+			onPress={() => {
+				goToPage?.(index);
+			}}
+			style={({ pressed }: { pressed: boolean }) => ({
+				opacity: pressed ? 0.7 : 1
+			})}>
+			<View style={styles.tabBarItem}>
+				<Emoji
+					content={tab.emoji}
+					standardEmojiStyle={standardEmojiStyle}
+					customEmojiStyle={customEmojiStyle}
+					baseUrl={baseUrl}
+					getCustomEmoji={getCustomEmoji}
+				/>
+				<Text style={[styles.reactionCount, { color: colors.auxiliaryTintColor }]}>{tab.usernames.length}</Text>
+			</View>
+			<View
+				style={[
+					styles.tabLine,
+					activeTab === index
+						? { backgroundColor: colors.tintColor, height: 2 }
+						: {
+								backgroundColor: colors.separatorColor,
+								height: 1
+						  }
+				]}
+			/>
+		</Pressable>
+	);
+};
+
+const ReactionsTabBar = ({ tabs, activeTab, goToPage, baseUrl, getCustomEmoji }: IReactionsTabBar) => (
+	<View>
+		<FlatList
+			data={tabs}
+			keyExtractor={item => item.emoji}
+			horizontal
+			showsHorizontalScrollIndicator={false}
+			renderItem={({ item, index }) => (
+				<TabBarItem
+					tab={item}
+					index={index}
+					activeTab={activeTab}
+					goToPage={goToPage}
+					baseUrl={baseUrl}
+					getCustomEmoji={getCustomEmoji}
+				/>
+			)}
+		/>
+	</View>
+);
+
+const UsersList = ({ tabLabel }: { tabLabel: IReaction }) => {
+	const { colors } = useTheme();
+	const { emoji, usernames } = tabLabel;
+	return (
+		<View>
+			<View style={styles.emojiName}>
+				<Text style={{ color: colors.auxiliaryTintColor }}>{emoji}</Text>
+			</View>
+			<FlatList
+				data={usernames}
+				renderItem={({ item }: { item: string }) => (
+					<View style={styles.userItemContainer}>
+						<Avatar text={item} size={36} />
+						<View style={styles.usernameContainer}>
+							<Text style={[styles.usernameText, { color: colors.titleText }]}>{item}</Text>
+						</View>
+					</View>
+				)}
+				keyExtractor={item => item}
+			/>
+		</View>
+	);
+};
+
+const ReactionsList = ({ reactions, baseUrl, getCustomEmoji }: IReactionsListProps): React.ReactElement => {
+	// sorting reactions in descending order on the basic of number of users reacted
+	const sortedReactions = reactions?.sort((reaction1, reaction2) => reaction2.usernames.length - reaction1.usernames.length);
+
+	return (
+		<View style={styles.reactionsListContainer}>
+			<ScrollableTabView renderTabBar={() => <ReactionsTabBar baseUrl={baseUrl} getCustomEmoji={getCustomEmoji} />}>
```

Comment:
Do you know if there is a way to improve the gesture of this?
On ios it's pretty bad to use, the scroll doesn't work properly.